### ISSUE TITLE
Configure RedPanda to use the AWS Avro Serializer

### ISSUE THREAD

**Issue by francescomucio** [redpanda-data/console#854]
We are investigating the possibility to use Redpanda on top of our AWS MSK cluser where we publish Avro events.
We learned the hard way that MSK doesn't ingest normal Avro events, but they needs to serialized with the custom AWS serializer (`com.amazonaws.services.schemaregistry.serializers.avro.AWSKafkaAvroSerializer`). 

Do we need to have a custom serializer if we use Redpanda with Avro events in MSK? And in case, is this a possibility?

**Comment by francescomucio** [comment]
Will it be possible to contribute to this repo adding this serializer?

**Comment by weeco** [comment]
Yes, ideally take a look at this: https://github.com/redpanda-data/console/tree/improve-message-viewer/backend/pkg/serde . This is a refactoring of the deserialization code which we plan to merge within the next few weeks. This should also make it easier to add a new deserialization strategy
